lighthouse-core/report/v2/report-styles.css

```diff
@@ -73,6 +73,10 @@ a {
   color: #0c50c7;
 }
 
+summary:active {
+  outline: none;
```

Comment:
The browser's default outlines are hideous,  but they're important for a11y :(

One thing we could try is using the green bar for `:active`, but nuking the outline. That way there's a visual indication but it's a bit nicer.

cc @robdodson